dss/operations/storage.py

```diff
@@ -146,12 +158,16 @@ def process_key(self, key):
                     update_aws_content_type(client, self.replica.bucket, blob_key, file_metadata['content-type'])
                 elif Replica.gcp == self.replica:
                     update_gcp_content_type(client, self.replica.bucket, blob_key, file_metadata['content-type'])
+            elif Replica.aws == self.replica:
+                blob_etag = self.handle.get_cloud_checksum(self.replica.bucket, blob_key)
+                if blob_etag != file_metadata['s3-etag']:
+                    update_aws_content_type(client, self.replica.bucket, blob_key, file_metadata['content-type'])
```

Comment:
Should this `assert` that the new etag matches the metadata etag on the next line?

Reply:
will add